packages/sirius-web/frontend/sirius-web-application/src/views/new-project/NewProjectView.types.ts

```diff
@@ -10,8 +10,16 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
+
+import { GQLProjectTemplate } from '../project-browser/create-projects-area/useProjectTemplates.types';
+
 export interface NewProjectViewState {
   name: string;
+  pristineName: boolean;
+  availableTemplates: GQLProjectTemplate[] | null;
+  templateSelectionOpen: boolean;
   librariesImportOpen: boolean;
+  selectedTemplate: string;
   selectedLibraries: string[];
+  creating: boolean;
```

Comment:
I suspect that we should probably use the loading value from `useCreateProject()` instead of maintaining our own here.